### PR TITLE
Oc 429 remove activate device request

### PR DESCRIPTION
**Pull request by smvdheijden**

### CODE REVIEW

**Comment by bvdzwet** [comment]
In cucumber-tests-platform-common there is a folder soap-ui-project that has Admin-SoapUI-project,xml that has content referring to ActivateDeviceRequest, DeactivateDeviceRequest and their responses.

On the topic of SOAP UI projects, there are also SOAP UI projects in the config repos with references to the removed requests and responses.

The Documentation repo has documentation of these operations that had better be removed as well.

File: cucumber-tests-platform-common/src/test/java/com/alliander/osgp/cucumber/platform/common/glue/steps/ws/admin/devicemanagement/DeactivateDeviceSteps.java [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-
-import cucumber.api.java.en.Then;
-import cucumber.api.java.en.When;
-
-public class DeactivateDeviceSteps extends GlueBase {
-
-    @Autowired
-    private AdminDeviceManagementClient client;
-
-    @When("^receiving a deactivate device request$")
```
Comment (bvdzwet) [inline cucumber-tests-platform-common/src/test/java/com/alliander/osgp/cucumber/platform/common/glue/steps/ws/admin/devicemanagement/DeactivateDeviceSteps.java]: It looks like this step is still referenced from `DeactivateDevice.feature` in cucumber-tests-platform-common. Shouldn't that feature be removed?

File: cucumber-tests-platform-common/src/test/resources/features/common/basic-osgp-functions/AuthorizePlatformFunctions.feature [hunk 2]
```diff
@@ -68,5 +68,3 @@ Feature: BasicOsgpFunctions Authorizing Platform Functions
       | CHANGE_FIRMWARE        | USER                  | false   |
       | REMOVE_FIRMWARE        | ADMIN                 | true    |
       | REMOVE_FIRMWARE        | USER                  | false   |
-      | ACTIVATE_DEVICE        | ADMIN                 | true    |
-      | ACTIVATE_DEVICE        | USER                  | false   |
```
Comment (bvdzwet) [inline cucumber-tests-platform-common/src/test/resources/features/common/basic-osgp-functions/AuthorizePlatformFunctions.feature]: Shouldn't `DEACTIVATE_DEVICE` (lines 49, 50) be removed from the AuthorizePlatformFunctions feature as well?

Comment (smvdheijden) [reply cucumber-tests-platform-common/src/test/resources/features/common/basic-osgp-functions/AuthorizePlatformFunctions.feature]: Yes, removed the lines

File: cucumber-tests-platform-common/src/test/java/com/alliander/osgp/cucumber/platform/common/glue/steps/ws/admin/devicemanagement/DeactivateDeviceSteps.java (hunk 1 above)
Comment (smvdheijden) [reply cucumber-tests-platform-common/src/test/java/com/alliander/osgp/cucumber/platform/common/glue/steps/ws/admin/devicemanagement/DeactivateDeviceSteps.java]: Removed feature file

**Pull request merged**